pyrasterframes/src/main/python/requirements.txt

```diff
@@ -3,3 +3,4 @@ shapely
 pyspark==2.3.3
 numpy>=1.7
 pandas
+matplotlib<3.0.0  # no python 2.7 support after v2.x.x
```

Comment:
What happens in our 2.7 environment? Does this run OK in CircleCI?

Reply:
I think the build passes with this requirement.